services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/spec_dynamic_sidecar.py

```diff
@@ -40,6 +43,20 @@ def _get_dy_sidecar_env_vars(
         "POSTGRES_USER": f"{app_settings.POSTGRES.POSTGRES_USER}",
         "POSTGRES_DB": f"{app_settings.POSTGRES.POSTGRES_DB}",
         "STORAGE_ENDPOINT": app_settings.STORAGE_ENDPOINT,
+        "REGISTRY_AUTH": str(registry_settings.REGISTRY_AUTH),
+        "REGISTRY_PATH": str(registry_settings.REGISTRY_PATH),
+        "REGISTRY_URL": str(registry_settings.REGISTRY_URL),
+        "REGISTRY_USER": str(registry_settings.REGISTRY_USER),
+        "REGISTRY_PW": str(registry_settings.REGISTRY_PW.get_secret_value()),
+        "REGISTRY_SSL": str(registry_settings.REGISTRY_SSL),
+        "RABBIT_HOST": str(rabbit_settings.RABBIT_HOST),
+        "RABBIT_PORT": str(rabbit_settings.RABBIT_PORT),
+        "RABBIT_USER": str(rabbit_settings.RABBIT_USER),
+        "RABBIT_PASSWORD": str(rabbit_settings.RABBIT_PASSWORD.get_secret_value()),
+        "RABBIT_CHANNELS": json.dumps(rabbit_settings.RABBIT_CHANNELS),
```

Comment:
Should start using new ``json_dumps`` from ``servicelib.json_serialization`` which implements all pydantic encoders

Reply:
used the one from json_serialization